code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -200,6 +200,22 @@
 #undef GEAR_UNFASTENED
 #undef GEAR_LOOSE
 
+/obj/structure/window/reinforced/clockwork
+	name = "ratvarian window"
+	desc = "A reinforced yellow-glass window, gently luminous."
+	icon = 'icons/obj/clockwork_objects.dmi'
+
+/obj/structure/window/reinforced/clockwork/New()
+	..()
+	spawn(1)
```

Comment:
If you give it some arguments in New() you won't need to use a spawn(1) and I'd recommend doing so